finesse/device_info.py

```diff
@@ -10,10 +11,10 @@ class DeviceParameter:
     name: str
     """Name for the parameter."""
 
-    possible_values: list[str]
+    possible_values: Sequence[Any]
     """Possible values the parameter can take."""
 
-    default_value: str | None = None
+    default_value: Any | None = None
```

Comment:
This feels odd. Doesn't `Any` include `None`?

Reply:
True! I'll fix it up